### PR TITLE
More content changes to FR and EN but not content pages

### PR DESCRIPTION
**Pull request by sarayek**
### Description

List of proposed changes:

-
-

### Additional Notes

### CODE REVIEW

File: frontend/public/locales/en/quiz.json [hunk 1]
```diff
@@ -91,8 +91,8 @@
     },
     "question-where": {
       "option-canada-ft": "In Canada",
-      "option-canada-pt-60-or-more": "In Canada <span>more</span> than 6 months a year",
-      "option-canada-pt-less-than-60": "In Canada <span>less</span> than 6 months a year",
+      "option-canada-pt-60-or-more": "In Canada more than 6 months a year",
+      "option-canada-pt-less-than-60": "In Canada less than 6 months a year",
```
Comment (gregory-j-baker) [inline frontend/public/locales/en/quiz.json:95]: If you remove the `<span>` here, then you also have to modify the associated lines in `src/components/quiz/questions/QuestionWhere.tsx`.

If you're not comfortable doing that, I would suggest to leave the `<span>` in and ask one of the developers to remove it.

File: frontend/public/locales/fr/learn.json [hunk 2]
```diff
@@ -28,11 +28,11 @@
...
       "cards": {
         "from-work-to-retirement": {
-          "body": "Vous pouvez choisir différentes manière d'effectuer une transition du monde du travail a celui de la retraite. Prendre sa retraite du monde du travail et commencer a recevoir ses pensions publiques sont deux choses différentes.",
+          "body": "Vous pouvez choisir différentes manières d'effectuer une transition du monde du travail à celui de la retraite. Prendre sa retraite du monde du travail et commencer à recevoir ses pensions publiques sont deux choses différentes.",
           "title": "Transition du monde du travail à la retraite"
         },
         "rules-of-thumb": {
-          "body": "Jetez un coup d'œil sur ces conseils pour vous aider a prendre la décision de quand commencer à recevoir vos pensions. Faite le choix adapter à votre situation.",
+          "body": "Jetez un coup d’œil à ces conseils pour vous aider à choisir quand commencer à recevoir vos pensions publiques. Faites le choix adapté à votre situation.",
```
Comment (gregory-j-baker) [inline frontend/public/locales/fr/learn.json:35]:
```suggestion
          "body": "Jetez un coup d'œil à ces conseils pour vous aider à choisir quand commencer à recevoir vos pensions publiques. Faites le choix adapté à votre situation.",
```

(to remain consistent with the rest of the application)

File: frontend/public/locales/fr/quiz.json [hunk 3]
```diff
@@ -91,8 +91,8 @@
     },
     "question-where": {
       "option-canada-ft": "Au Canada",
-      "option-canada-pt-60-or-more": "Au Canada <span>plus</span> de 6 mois par année",
-      "option-canada-pt-less-than-60": "Au Canada <span>moins</span> de 6 mois par année",
+      "option-canada-pt-60-or-more": "Au Canada plus de 6 mois par année",
+      "option-canada-pt-less-than-60": "Au Canada moins de 6 mois par année",
```
Comment (gregory-j-baker) [inline frontend/public/locales/fr/quiz.json:95]: If you remove the `<span>` here, then you also have to modify the associated lines in `src/components/quiz/questions/QuestionWhere.tsx`.

If you're not comfortable doing that, I would suggest to leave the `<span>` in and ask one of the developers to remove it.

File: frontend/src/data/tasks.json [hunk 4]
```diff
@@ -1960,7 +1960,7 @@
...
+        "descriptionFr": "Il est normal de ne pas savoir quand vous voulez prendre votre retraite ! Choisir de se retirer du monde du travail et recevoir ses pensions publiques sont deux choses distinctes. Cependant, il est important de décider quand vous voulez recevoir vos pensions publiques. Avoir un plan avant de prendre sa retraite peut vous procurer la tranquillité d'esprit.\n\nVous n'avez pas besoin d'arrêter de travailler pour demander vos pensions. Le RPC et la SV vous permettent de choisir quand commencer votre pension de retraite, et chaque mois que vous reportez augmente votre montant mensuel. Le montant mensuel le plus élevé que vous pouvez recevoir est atteint à 70 ans, après quoi il n'y a aucun avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à recevoir votre pension de retraite du RPC dès l'âge de 60 ans, mais avec une réduction permanente. Pour toucher votre pension de la SV, vous devez être âgé d'au moins 65 ans.\n\nSi vous avez un faible revenu ou une mauvaise santé, vous devriez commencer lorsque vous êtes admissible.",
```
Comment (gregory-j-baker) [inline frontend/src/data/tasks.json:1963]:
```suggestion
        "descriptionFr": "Il est normal de ne pas savoir quand vous voulez prendre votre retraite! Choisir de se retirer du monde du travail et recevoir ses pensions publiques sont deux choses distinctes. Cependant, il est important de décider quand vous voulez recevoir vos pensions publiques. Avoir un plan avant de prendre sa retraite peut vous procurer la tranquillité d'esprit.\n\nVous n'avez pas besoin d'arrêter de travailler pour demander vos pensions. Le RPC et la SV vous permettent de choisir quand commencer votre pension de retraite, et chaque mois que vous reportez augmente votre montant mensuel. Le montant mensuel le plus élevé que vous pouvez recevoir est atteint à 70 ans, après quoi il n'y a aucun avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à recevoir votre pension de retraite du RPC dès l'âge de 60 ans, mais avec une réduction permanente. Pour toucher votre pension de la SV, vous devez être âgé d'au moins 65 ans.\n\nSi vous avez un faible revenu ou une mauvaise santé, vous devriez commencer lorsque vous êtes admissible.",
```

(removed the space before the exclamation point -- see https://www.btb.termiumplus.gc.ca/tcdnstyl-chap?lang=fra&lettr=indx96&info0=17.07)

File: frontend/src/data/tasks.json [hunk 5]
```diff
@@ -1977,7 +1977,7 @@
...
```
Comment (gregory-j-baker) [inline frontend/src/data/tasks.json:1980]:
```suggestion
        "descriptionFr": "Il est normal de ne pas savoir quand vous voulez prendre votre retraite! Choisir de se retirer du monde du travail et recevoir ses pensions publiques sont deux choses distinctes. Cependant, il est important de décider quand vous voulez recevoir vos pensions publiques. Avoir un plan avant de prendre sa retraite peut vous procurer la tranquillité d'esprit.\n\nVous n'avez pas besoin d'arrêter de travailler pour demander vos pensions. Le RPC et la SV vous permettent de choisir quand commencer votre pension de retraite, et chaque mois que vous reportez augmente votre montant mensuel. Le montant mensuel le plus élevé que vous pouvez recevoir est atteint à 70 ans, après quoi il n'y a aucun avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à recevoir votre pension de retraite du RPC dès l'âge de 60 ans, mais avec une réduction permanente. Pour toucher votre pension de la SV, vous devez être âgé d'au moins 65 ans.\n\nSi vous avez un faible revenu ou une mauvaise santé, vous devriez commencer lorsque vous êtes admissible.",
```

(removed the space before the exclamation point -- see https://www.btb.termiumplus.gc.ca/tcdnstyl-chap?lang=fra&lettr=indx96&info0=17.07)

File: frontend/public/locales/fr/quiz.json (hunk 3 above)
Comment (sarayek) [reply frontend/public/locales/fr/quiz.json:95]: got it - will ask Frank

Comment (gregory-j-baker) [reply frontend/public/locales/fr/quiz.json:95]: 👍🏻 In that case, please add the `<span>` elements back in and  Frank can remove them in a future commit, along with the associated lines in `QuestionWhere.tsx`.

Comment (gregory-j-baker) [reply frontend/public/locales/fr/quiz.json:95]: Did you add the `<span>` back in both places? I still don't see it in the french quiz.json.

> I made the change in my fork - do I need to put in another PR?

No, your PR will update as you update your fork's branch.

Comment (sarayek) [reply frontend/public/locales/fr/quiz.json]: done - updated in EN and FR

**Review by gregory-j-baker — APPROVED** [review]

**Comment by gregory-j-baker** [comment]
Merged!

**Pull request merged**